model-optimizer/mo/main.py

```diff
@@ -254,9 +268,24 @@ def emit_ir(graph: Graph, argv: argparse.Namespace):
 
     if not (argv.framework == 'tf' and argv.tensorflow_custom_operations_config_update):
         output_dir = argv.output_dir if argv.output_dir != '.' else os.getcwd()
-        print('\n[ SUCCESS ] Generated IR version {} model.'.format(get_ir_version(argv)))
-        print('[ SUCCESS ] XML file: {}.xml'.format(os.path.join(output_dir, argv.model_name)))
-        print('[ SUCCESS ] BIN file: {}.bin'.format(os.path.join(output_dir, argv.model_name)))
+        orig_model_name = os.path.normpath(os.path.join(output_dir, argv.model_name))
+
+        # this try-except is just an additional reinsurance to be sure that
+        # IE dependency search doesn't break MO pipeline
+        try:
+            if find_ie_version(silent=True):
+                path_to_offline_transformations = os.path.join(os.path.realpath(os.path.dirname(__file__)), 'back',
+                                                               'offline_transformations.py')
+                status = subprocess.run([sys.executable, path_to_offline_transformations, orig_model_name], env=os.environ, timeout=100)
+                if status.returncode != 0 and not argv.silent:
+                    print("[ WARNING ] offline_transformations return code {}".format(status.returncode))
+        except Exception as e:
+            # TODO: send error message
+            pass
+
+        print('[ SUCCESS ] Generated IR version {} model.'.format(get_ir_version(argv)))
```

Comment:
We don't perform any transformations on produced IR. The main idea is to check that all IE dependency works.
But when we will use SRS technique then the message that IR was generated must be exactly after transformations execution because IR that was generated before transformations is intermediate, temporary and will be replace with new one. But in this PR we don't do that.